src/components/barchart/BarChart.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import {
+  BarDatum,
+  BarTooltipProps,
+  ComputedDatum,
+  ResponsiveBar,
+  ResponsiveBarSvgProps,
+} from '@nivo/bar'
+import { AxisProps, AxisLegendPosition } from '@nivo/axes'
+import { appTheme } from '../../appTheme'
+import { Margin } from '@nivo/core'
+
+export type Props<D> = ResponsiveBarSvgProps<D & BarDatum> & {
```

Comment:
I think there's an opportunity to simplify this bar chart, to accept an array of `{ id: , value: }` objects. Since we haven't used this yet in more than one place, I wanted to leave things flexible while we work out the kinks.